apps/backend/src/modules/project-groups/project-group.repository.ts

```diff
@@ -27,24 +31,21 @@ class ProjectGroupRepository implements Repository {
 				key,
 				name,
 				permissions,
-				projects: projectId,
+				projects: [projectId],
 				users,
 			};
 
 			const createdProjectGroup = await this.projectGroupModel
 				.query(trx)
 				.insertGraph(projectGroupData, { relate: true })
 				.returning("*")
-				.withGraphJoined("[permissions, projects, users]");
+				.withGraphFetched("[permissions, projects, users]");
```

Comment:
in this case, we using Objection.js, and .execute() is unnecessary because the query methods themselves (like .insertGraph()) already execute the query when called.